File: king-admin-java/src/main/java/com/oukingtim/web/TbTodoController.java
package com.oukingtim.web;


import com.baomidou.mybatisplus.mapper.EntityWrapper;
import com.oukingtim.domain.TbTodo;
import com.oukingtim.service.TbTodoService;
import com.oukingtim.util.ShiroUtils;
import com.oukingtim.web.vm.ResultVM;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.List;

/**
 * Created by oukingtim
 */
@RestController
@RequestMapping("/sys/todo")
public class TbTodoController extends BaseController<TbTodoService, TbTodo> {

    @GetMapping("/getlist")
    public ResultVM getList() {
        TbTodo tbTodo = new TbTodo();
        tbTodo.setCreateUserId(ShiroUtils.getUserId());
        List<TbTodo> list = service.selectList(new EntityWrapper<>(tbTodo));
        return ResultVM.ok(list);
    }
}

File: king-admin-kotlin/src/main/kotlin/com/oukingtim/web/TbTodoController.kt
package com.oukingtim.web

import com.baomidou.mybatisplus.mapper.EntityWrapper
import com.oukingtim.domain.TbTodo
import com.oukingtim.service.TbTodoService
import com.oukingtim.util.ShiroUtils
import com.oukingtim.web.vm.ResultVM
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

/**
 * Created by oukingtim
 */
@RestController
@RequestMapping("/sys/todo")
class TbTodoController : BaseController<TbTodoService, TbTodo>(){

    @GetMapping("/getlist")
    fun getList(): ResultVM {
        val tbTodo = TbTodo()
        tbTodo.createUserId=ShiroUtils.getUserId()
        val list = service!!.selectList(EntityWrapper(tbTodo))
        return ResultVM.ok(list)
    }
}